src/test/ui/chalkify/type_inference.stderr

```diff
@@ -1,11 +1,11 @@
 error[E0277]: the trait bound `{float}: Bar` is not satisfied
-  --> $DIR/type_inference.rs:27:5
+  --> $DIR/type_inference.rs:27:14
    |
 LL | fn only_bar<T: Bar>(_x: T) { }
    |                --- required by this bound in `only_bar`
 ...
 LL |     only_bar(x);
-   |     ^^^^^^^^ the trait `Bar` is not implemented for `{float}`
+   |              ^ the trait `Bar` is not implemented for `{float}`
```

Comment:
It's a bit weird this output changed.

Reply:
It is being changed to what the current stable does, see for example [this playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=54af3f56980ba9ffa14752222c0bb84c).